wp-content/plugins/core/src/Assets/Block_Assets_Enqueuer.php

```diff
@@ -0,0 +1,77 @@
+<?php declare(strict_types=1);
+
+namespace Tribe\Plugin\Assets;
+
+use Tribe\Plugin\Blocks\Blocks_Definer;
+
+class Block_Assets_Enqueuer extends Assets_Enqueuer {
```

Comment:
@dpellenwood @ChrisMKindred @JMRhodes just an FYI that I temporarily ported this over from LendNation so I'm able to see my styles as I work through core blocks. I know BE will be looking into a better implementation soon.